docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -93,91 +102,114 @@ const session = new Session({ hostname: "jsonplaceholder.typicode.com"});
 })();
 ```
 
-### Checkpoint one
+### Verify your build with the programmatic API **[correct?]**
+
+In your terminal, issue `npm run build` in your terminal to verify a clean compilation and confirm that no lint errors are present.
 
-Issue `npm run build` to verify a clean compilation and confirm that no lint errors are present. At this point in this tutorial, you have a programmatic API
-that will be used by your handler or another Node.js application. Next you'll define the command syntax for the command that will use your programmatic Node.js APIs.
+At this point, you have a programmatic API to be used by your handler or another Node.js application. Next, define the command syntax for the command that uses your programmatic Node.js APIs.
```

Comment:
Change "... you have a programmatic API to be used ..." to "... you have a programmatic API that can be used ..."